src/test/java/org/cactoos/io/WriterAsOutputTest.java

```diff
@@ -57,22 +56,20 @@ public void writesLargeContentToFile() throws IOException {
         try (final OutputStreamWriter writer = new OutputStreamWriter(
             new FileOutputStream(temp.toFile()), StandardCharsets.UTF_8
         )) {
-            MatcherAssert.assertThat(
+            new Assertion<>(
                 "Can't copy Input to Output and return Input",
-                new TextOf(
-                    new TeeInput(
-                        new ResourceOf("org/cactoos/large-text.txt"),
-                        new WriterAsOutput(writer)
+                () -> new TextOf(
+                    new Sticky(
```

Comment:
@Iprogrammerr Is necessary use the Sticky decorator here?

Reply:
@fabriciofx Same as above.

Reply:
@Iprogrammerr the same above.